cubic-server/Default/DefaultWorld.cpp

```diff
@@ -23,3 +26,15 @@ void DefaultWorld::initialize()
     _theNether->initialize();
     _overworld->initialize();
 }
+
+void DefaultWorld::updateTime() {
+    std::shared_ptr<std::vector<uint8_t>> data;
+    // add tick to age of the world and time
+    _age += 1;
+    _time += 1;
+    if (_time > 24000)
+        _time = 0;
+
+    // send packets to clients (missing clients in architecture)
+    data = protocol::createUpdateTime({_age, _time});
```

Comment:
The clients will be added by #48 , so maybe we can wait for that to be merged before merging this